Model/src/GamePackage/Player.java

```diff
@@ -1,4 +1,265 @@
 package GamePackage;
 
+import java.util.ArrayList;
+
+import BoardPackage.BoardSquare;
+import BoardPackage.PropertySquare;
+import CardPackage.Card;
+
+/**********************************************************************
+ * The Player class will keep track of each player's money, properties,
+ * and other stats.
+ * 
+ * @author Dustin Ecker
+ * @version 2/11/2018
+ *********************************************************************/
 public class Player {
+	
+	/** Player's name */
+	private String displayName;
+	
+	/** Player's token */
+	private String token;
+	
+	/** Tells if player can move */
+	private boolean canMove;
+	
+	/** Tells if player is bankrupt */
+	private boolean isBankrupt;
+	
+	/** Holds player's current wallet balance */
+	private int wallet;
+	
+	/** ArrayList that holds player's owned properties */
+	private ArrayList<BoardSquare> propertiesOwned;
+	
+	/** ArrayList that holds player's cards held */
+	private ArrayList<Card> cardsHeld;
+	
+	/******************************************************************
+	 * Constructor method for Player class.
+	 * 
+	 * @param displayName Sets the player's name.
+	 * @param token Sets the player's chosen token.
+	 * @param wallet Sets the player's starting wallet value.
+	 *****************************************************************/
+	public Player(String displayName, String token, int wallet) {
+		super();
```

Comment:
Should it be super(name); and have name as a param as well?

Reply:
See my review.

Reply:
Yeah, disregard this comment! ;)